packages/frontend/src/components/common/SpaceActionModal/CreateSpaceAddUser.tsx

```diff
@@ -0,0 +1,188 @@
+                                            {user.firstName} {user.lastName}
+                                        </PrimaryText>
+
+                                        <SecondaryText>
+                                            {user.email}
+                                        </SecondaryText>
+                                    </>
+                                ) : (
+                                    <PrimaryText $selected={isSelected}>
+                                        {user.email}
+                                    </PrimaryText>
+                                )}
+                            </PrimaryAndSecondaryTextWrapper>
+                        </FlexWrapper>
+                    }
+                    onClick={(e) => {
+                        // Toggle user selected if selected
+                        if (usersSelected.includes(user.userUuid))
+                            setUsersSelected(
+                                usersSelected.filter(
+                                    (uuid) => uuid !== user.userUuid,
+                                ),
+                            );
+                        else handleClick(e);
+
+                        setSearchQuery('');
+                    }}
+                    shouldDismissPopover={false}
+                />
+            );
+        },
+        [usersSelected, organizationUsers],
+    );
+
+    return (
+        <>
+            <FlexWrapper>
+                <MultiSelect2
+                    fill
+                    itemPredicate={filterUser}
+                    itemRenderer={renderUserShare}
+                    items={userUuids || []}
+                    noResults={<MenuItem2 disabled text="No suggestions." />}
+                    onItemSelect={(select) => {
+                        setUsersSelected([...usersSelected, select]);
+                        setSearchQuery('');
+                    }}
+                    query={searchQuery}
+                    onQueryChange={setSearchQuery}
+                    tagRenderer={(userUuid) =>
+                        getUserNameOrEmail(userUuid, organizationUsers)
+                    }
+                    resetOnQuery={true}
+                    popoverProps={{
+                        minimal: true,
+                        matchTargetWidth: true,
+                        onClosing: () => setSearchQuery(''),
+                    }}
+                    tagInputProps={{
+                        placeholder: 'Start typing to search for users...',
+                        addOnBlur: false,
+                        tagProps: {
+                            minimal: true,
+                        },
+                        onRemove: (e) => {
+                            setSearchQuery('');
+                            handleRemove(e);
+                        },
+                    }}
+                    selectedItems={usersSelected}
+                />
+            </FlexWrapper>
+        </>
```

Comment:
can you also remove this extra fragment wrapper. since we have 4space formatting in the code it becomes mushed together in the right side.